spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      before_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = before_locale
```

Comment:
So, for this particular test, I think it's important that we put the environment variable back after changing it, since that DOES bleed into other tests, even in completely different describe blocks, unexpectedly. But it might be overkill to do it to all of them.

I'm willing to prepend the locale change to the `msgcmp` command as long as it doesn't cause breakages on most normal operating systems. I know we're only using this internally on Linux right now, but it's just good practice for and open source tool being used in lots of environments.